test/e2e/resource_quota.go

```diff
@@ -258,6 +259,19 @@ var _ = framework.KubeDescribe("ResourceQuota", func() {
 		pod, err = f.Client.Pods(f.Namespace.Name).Create(pod)
 		Expect(err).To(HaveOccurred())
 
+		By("Ensuring a pod cannot update its resource requirements")
```

Comment:
This seems like more of an int test to me, but I'm okay with the e2e exercising it too